src/openforms/submissions/models/submission.py

```diff
@@ -297,6 +299,20 @@ def save_registration_status(self, status, result):
 
         self.save(update_fields=update_fields)
 
+    @property
+    def total_configuration_wrapper(self):
+        if not self._total_configuration_wrapper:
+            state = self.load_execution_state()
+            form_steps = state.form_steps
+            if len(form_steps) == 0:
+                return
+
+            wrapper = copy.deepcopy(form_steps[0].form_definition.configuration_wrapper)
```

Comment:
it'd probably make more sense to instantiate `FormioConfigurationWrapper` and treat it as a separate object used in this particular scenario, rather than have it cached on the `form_definition`:

```
wrapper = FormioConfigurationWrapper(form_steps[0].form_definition.configuration_wrapper)
```